.github/workflows/build.yml

```
@@ -21,97 +21,97 @@ jobs:
-        content=${response::-3}
-
-        if [ $http_code -eq 200 ]; then
-            run_id=$(echo "$content" | jq -r '.id')
-            echo "Pipeline triggered successfully. Run ID: $run_id"
-        else
-            echo "Failed to trigger pipeline. HTTP Status Code: $http_code"
-            echo "Response: $content"
-            exit 1
-        fi
+    # - name: Trigger Azure DevOps Pipeline
+    #   env:        
```

yamllint

[error] trailing spaces

(trailing-spaces)